RELEASES.md

```diff
@@ -1,3 +1,105 @@
+Version 1.69.0 (2023-04-20)
+==========================
+
+<a id="1.69.0-Language"></a>
+
+Language
+--------
+
+- [Deriving built-in traits on packed structs works with `Copy` fields.](https://github.com/rust-lang/rust/pull/104429/)
+- [Stabilize the `cmpxchg16b` target feature on x86 and x86_64.](https://github.com/rust-lang/rust/pull/106774/)
+- [Improve analysis of trait bounds for associated types.](https://github.com/rust-lang/rust/pull/103695/)
+- [Allow associated types to be used as union fields.](https://github.com/rust-lang/rust/pull/106938/)
+- [Allow `Self: Autotrait` bounds on dyn-safe trait methods.](https://github.com/rust-lang/rust/pull/107082/)
+- [Treat `str` as containing `[u8]` for auto trait purposes.](https://github.com/rust-lang/rust/pull/107941/)
+
+<a id="1.69.0-Compiler"></a>
+
+Compiler
+--------
+
+- [Upgrade mingw-w64 on CI to GCC 12.3.](https://github.com/rust-lang/rust/pull/100178/)
```

Comment:
Maybe something more detailed?
```suggestion
- [Upgrade mingw-w64 on CI to version v10 and GCC 12.3.](https://github.com/rust-lang/rust/pull/100178/)
```

Reply:
Sure, but "version v10" seems to be redundant "v"s?

Reply:
Dunno, they call their versions with `v` on the official page: https://www.mingw-w64.org/changelog
For me it doesn't really matter but tried to be closer to the scheme used by the website.

Reply:
How about:
```suggestion
- [Upgrade `*-pc-windows-gnu` on CI to mingw-w64 v10 and GCC 12.3.](https://github.com/rust-lang/rust/pull/100178/)
```

Also, should that be GCC 12.2? The `MINGW_ARCHIVE` names look like it, but I'm not sure where I got 12.3.

Reply:
Yeah, looks better.

> Also, should that be GCC 12.2? The `MINGW_ARCHIVE` names look like it, but I'm not sure where I got 12.3.

Good catch! This definitely should be GCC 12.2.

Reply:
```suggestion
- [Upgrade `*-pc-windows-gnu` on CI to mingw-w64 v10 and GCC 12.2.](https://github.com/rust-lang/rust/pull/100178/)
```